### ISSUE TITLE
Web Remote plugin

### ISSUE THREAD

**Issue by mattsigal** [BiglySoftware/BiglyBT#26]
Testing the app (in contrast to my usual Vuze/Transmission set-up), and it is looking good! I just have one comment and one question: 

1) When I imported my settings from Vuze, it did import and initialize the RSS feed plugin. However, it did *not* import all of my RSS queries/history/sources/et cetera. I had to go back into Vuze, run the RSS backup wizard, and then import that into BiglyBT. Not a huge inconvenience, but if BiglyBT could do that automatically in the future, would probably make someone happy.

2) I received a warning during the installation that one plugin was not available for BiglyBT - Vuze Web Remote (https://wiki.vuze.com/w/Vuze_Web_Remote). I actually use this plugin very frequently, and was wondering/hoping that it could be ported to BiglyBT?  I saw on the website that you have an Android app, but this was great because it is completely cross-platform.

**Comment by TuxPaper** [comment]
1. The installers from a week ago missed migrating the `%appdata%\Azureus\plugins\rssfeed\rssfeed.options` files, but the latest installers should have gotten that one.  Can you check the `%appdata%\Azureus\plugins\rssfeed\` to see if there's any other config-looking files there (like a .config or a .dat).  Maybe we are still missing one to migrate.  (Assuming you are on Windows.  Mac it's `~/Application Support/Vuze/plugins/rssfeed` and linux it's probably `~/.vuze/plugins/rssfeed`

2. We do have the Web Remote plugin!  So you probably hit another migration tool bug (ie. if it installer fails to connect to plugins.biglybt.com it will erroneously report that we don't have that plugin).  If you go to the very bottom of http://plugins.biglybt.com/ you will see the Web Remote plugin and you can install it from there (and it will probably keep your access code and settings from before)

**Comment by mattsigal** [comment]
1. Interesting! I had installed the app yesterday, using the most recent installer from the website. This was on Mac OS. I believe Vuze was set-up for multi-user support on my computer, so instead of being where you mentioned above, the config files are all stored in  /Users/Shared/Library/Application Support/Vuze/plugins/rssfeed

I just noticed that there is an Application Support/Vuze folder in my user directory library folder as well, but this was not the one actually being used by the application. Perhaps that was what caused the confusion.

2. That's great about the web remote plugin! I installed it manually from your plugins page and it seems to be working great.

Cheers!
